Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -489,6 +491,67 @@ def split_fields(fields: str = '', delimiter: str = ';') -> dict:
     return dic_fields
 
 
+def split_notes(raw_notes, note_type, time_info):
+    notes: List = []
+    notes_split = raw_notes.split('\n\n')
+    retrieved_last_note = False
+    for note in notes_split:
+        if not note:
+            continue
+        if 'Mirrored from Cortex XSOAR' in note:
+            if retrieved_last_note:  # add to last note only in case the note was not filtered by the time filter
+                notes[-1]['value'] += '\n\n Mirrored from Cortex XSOAR'
+            continue
+        note_info, note_value = note.split('\n')
+        created_on, created_by = note_info.split(' - ')
+        created_by = created_by.split(' (')[0]
+
+        # convert note creation time to UTC
+        created_on_UTC = datetime.strptime(created_on, DATE_FORMAT) + time_info.get('timezone_offset')
```

Comment:
handle the aception here and return a more detailed error of what we failed to parse.